### PR TITLE
rsync: make rsync usable with dropbear

### PR DESCRIPTION
**Pull request by xeffyr**
Lets make rsync available for users which use dropbear instead of openssh.

### CODE REVIEW

File: packages/dropbear/build.sh [hunk 1]
```diff
@@ -15,13 +15,23 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" ac_cv_func_getpass=yes LIBS=-lreadline"
 # build a multi-call binary
 TERMUX_PKG_EXTRA_MAKE_ARGS="MULTI=1"
 
+termux_step_post_make_install() {
+    {
+        echo "#!$TERMUX_PREFIX/bin/sh"
+        echo "exec dbclient \"\${@}\""
```
Comment (tomty89) [inline packages/dropbear/build.sh]: Why not symlink?

Comment (xeffyr) [reply packages/dropbear/build.sh]: Ok, I will fix this. Just forgot that dropbearmulti understands 'ssh' symlink.

File: packages/dropbear/build.sh [hunk 2]
```diff
@@ -15,13 +15,19 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" ac_cv_func_getpass=yes LIBS=-lreadline"
 # build a multi-call binary
 TERMUX_PKG_EXTRA_MAKE_ARGS="MULTI=1"
 
+termux_step_post_make_install() {
+    ln -sf "${TERMUX_PREFIX}/bin/dropbearmulti" "${TERMUX_PREFIX}/bin/ssh"
```
Comment (tomty89) [inline packages/dropbear/build.sh]: I think we want to avoid absolute path in the target, so just `dropbearmulti`.

**Comment by fornwall** [comment]
Thanks! The updated packages are now available for installation.

**Pull request merged**